docs/manage/fleet/_index.md

```diff
@@ -117,3 +119,10 @@ If you are collaborating on a robot and controlling it using the [**Control** ta
 ### Deployment
 
 You and your collaborators can deploy [control logic](/program/apis/), [modular resources](/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](/services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
+
+## The Viam mobile app
```

Comment:
Let's also add a little glimpse of the app. ![ezgif com-optimize (6)](https://github.com/viamrobotics/docs/assets/5212232/5cedad94-801b-47e8-a6c1-e5b1a99d7ecf)

(Probably using the alignright css class)